cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -370,7 +373,7 @@ def string_action(self,component_name, data, blocking):
 		return ah
 
 
-#------------------- Move section -------------------#
+	#------------------- Move section -------------------#
```

Comment:
undo

Reply:
isn't this a typo? everything else has intendation

Reply:
the section delimiters have no indentation....see light section etc...